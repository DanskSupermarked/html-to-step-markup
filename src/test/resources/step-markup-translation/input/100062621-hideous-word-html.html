<p style="line-height: normal; margin: 0cm 0cm 0pt; mso-layout-grid-align: none;" class="MsoNormal"><span style="font-family: Arial;"><b style="mso-bidi-font-weight: normal;"><span style="font-size: 10pt;">Tendens 1 v&aelig;gmaling silkemat<!--?xml:namespace prefix = o ns = "urn:schemas-microsoft-com:office:office" /--></span></b></span><b style="mso-bidi-font-weight: normal;"><span style="font-family: 'Tahoma','sans-serif'; font-size: 10pt;"></span></b></p> <p style="line-height: normal; margin: 0cm 0cm 0pt; mso-layout-grid-align: none;" class="MsoNormal"><span style="font-family: Arial;"><span style="font-size: 10pt;">Akrylforst&aelig;rket, vandfortyndbar v&aelig;gmaling af h&oslash;j kvalitet med fremragende d&aelig;kkeevne. Er meget robust og super reng&oslash;ringsvenlig efter 30 dage. Lugtsvag og fri for opl&oslash;sningsmidler.</span></span><span style="font-family: 'Tahoma','sans-serif'; font-size: 10pt;"></span></p> <p style="line-height: normal; margin: 0cm 0cm 0pt; mso-layout-grid-align: none;" class="MsoNormal"><span style="font-family: Arial;"><span style="font-size: 10pt;">&nbsp;</span></span><span style="font-family: 'Tahoma','sans-serif'; font-size: 10pt;"></span></p> <p style="line-height: normal; margin: 0cm 0cm 0pt; mso-layout-grid-align: none;" class="MsoNormal"><span style="font-family: Arial;"><b style="mso-bidi-font-weight: normal;"><span style="font-size: 10pt;">Anvendelse: </span></b><span style="font-size: 10pt;">Anvendes indend&oslash;rs p&aring; grundede eller tidligere malede v&aelig;gge i stue, v&aelig;relse, entr&eacute;, k&oslash;kken, alrum og bryggers.</span></span><span style="font-size: 10pt;"><span style="font-size: 10pt;"> </span></span>&nbsp;</p> <p style="line-height: normal; margin: 0cm 0cm 0pt; mso-layout-grid-align: none;" class="MsoNormal">&nbsp;</p> <p style="line-height: normal; margin: 0cm 0cm 0pt; mso-layout-grid-align: none;" class="MsoNormal"><span style="font-family: Arial;"><b style="mso-bidi-font-weight: normal;"><span style="font-size: 10pt;">Forbehandling: </span></b><span style="font-size: 10pt;">Overfladen skal v&aelig;re t&oslash;r, fast, ren og afsmitningsfri. Snavsede og tidligere malede flader reng&oslash;res med malegrundrens og eftervaskes med rent vand. Smittende og sugende flader grundes med forankringsgrunder. </span></span><span style="font-family: 'Tahoma','sans-serif'; font-size: 10pt;"></span></p> <p style="line-height: normal; margin: 0cm 0cm 0pt; mso-layout-grid-align: none;" class="MsoNormal"><span style="font-family: Arial;"><span style="font-size: 10pt;">&nbsp;</span></span><span style="font-family: 'Tahoma','sans-serif'; font-size: 10pt;"></span></p> <p style="line-height: normal; margin: 0cm 0cm 0pt; mso-layout-grid-align: none;" class="MsoNormal"><span style="font-family: Arial;"><b style="mso-bidi-font-weight: normal;"><span style="font-size: 10pt;">P&aring;f&oslash;ring:</span></b><span style="font-size: 10pt;"> Malerrulle, syntetisk pensel eller airless spr&oslash;jte.<br /> </span></span><span style="font-family: 'Tahoma','sans-serif'; font-size: 10pt;"><br /> </span></p>
