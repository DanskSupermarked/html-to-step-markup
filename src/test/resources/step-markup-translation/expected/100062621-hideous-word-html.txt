<bold>Tendens 1 vægmaling silkemat</bold><return/>
<return/>
Akrylforstærket, vandfortyndbar vægmaling af høj kvalitet med fremragende dækkeevne. Er meget robust og super rengøringsvenlig efter 30 dage. Lugtsvag og fri for opløsningsmidler.<return/>
<return/>
<bold>Anvendelse:</bold> Anvendes indendørs på grundede eller tidligere malede vægge i stue, værelse, entré, køkken, alrum og bryggers.<return/>
<return/>
<bold>Forbehandling:</bold> Overfladen skal være tør, fast, ren og afsmitningsfri. Snavsede og tidligere malede flader rengøres med malegrundrens og eftervaskes med rent vand. Smittende og sugende flader grundes med forankringsgrunder.<return/>
<return/>
<bold>Påføring:</bold> Malerrulle, syntetisk pensel eller airless sprøjte.
